fast_build_docs.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
```

Comment:
This didn't work for me, I had to change it to:
```suggestion
#!/bin/bash
```